packages/docs/src/components/themes/Premium.vue

```diff
@@ -48,22 +48,26 @@
                 top
                 right
               >
-                <v-tooltip
-                  slot="activator"
-                  bottom
-                >
-                  <v-btn
-                    slot="activator"
-                    :href="template.demoUrl.length === 1 ? `${template.demoUrl[0]}?ref=vuetifyjs.com` : undefined"
-                    icon
-                    target="_blank"
-                    rel="noopener"
-                    aria-label="View Demo"
+                <template #activator="{ on: menu }">
+                  <v-tooltip
+                    bottom
+                    v-on="{ ...menu }"
```

Comment:
`v-on="menu"`? it's in few other places
also you can do `#activator="{ on }"` and `v-on="on"`, renaming is required only if you want to merge listeners from diffrents slots

Reply:
this was based off the example @KaelWD posted (where they are nested activators (menu -> tooltip). the syntax you provide doesn't work for this one. He also mentioned there was no need to spread when there is only 1 activator, which I had already went through and reverted them to `v-on="menu"`, though i think the `#activator="{ on }"` and `v-on="on"` syntax might be a little cleaner, so ill update them to that

Reply:
in this particular example, it should be `v-on="{ ...menu, ...tooltip }"` anyways